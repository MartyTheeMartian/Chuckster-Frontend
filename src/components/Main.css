.main-ctn {
  width: 100%;
  height: 100vh;
}
