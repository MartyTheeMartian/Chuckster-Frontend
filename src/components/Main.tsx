import './Main.css';

export default function Main(): JSX.Element {
  return (
    <div className='main-ctn has-background-info-light'>
      Main
    </div>
  );
}
