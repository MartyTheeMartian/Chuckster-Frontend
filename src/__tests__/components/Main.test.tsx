import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Main from 'components/Main';

it('renders App component', () => {
  const result = shallow(<Main />);
  expect(toJson(result)).toMatchSnapshot();
});
