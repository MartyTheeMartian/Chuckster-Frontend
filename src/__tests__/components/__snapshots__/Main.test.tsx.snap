// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders App component 1`] = `
<div
  className="main-ctn has-background-info-light"
>
  Main
</div>
`;
